lib/portage/tests/resolver/test_onlydeps_ideps.py

```diff
@@ -0,0 +1,93 @@
+# Copyright 2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+from portage.tests import TestCase
+from portage.tests.resolver.ResolverPlayground import (
+    ResolverPlayground,
+    ResolverPlaygroundTestCase,
+)
+
+
+class OnlydepsIdepsTestCase(TestCase):
```

Comment:
Can you add another set of tests where it's EAPI 7 to verify it makes no difference to the results?

It's partly about testing your change but also about providing a bit better EAPI/IDEPEND coverage.

Reply:
Yep. Will roll it in.